src/Storage/LaravelCacheStorage.php

```diff
@@ -43,8 +43,12 @@ public function fetch($key)
     public function save($key, CacheEntry $data)
     {
         try {
-            // getTTL returns seconds, Laravel needs minutes
-            $lifeTime = $data->getTTL() / 60;
+            if (app()->version() < '5.8.0') {
```

Comment:
This check is not correct. 😶
```php
php > echo '5.10.0' < '5.8.0';
1
```

Reply:
@grray Maybe we can use [version_compare](https://www.php.net/manual/fr/function.version-compare.php)?

Reply:
Yup, thanks for correction